Project/Core/Logger.cs

```diff
@@ -45,9 +40,27 @@ static Logger()
         /// </summary>
         public static Logger GetLogger => instance;
         // METHODS
-        private void Off(LogMode logMode)
+        /// <summary>
+        /// Turns off log modes.
+        /// <para/>
+        /// For multiple off modes use "|" between them, for example: <see cref="LogMode.Debug"/> | <see cref="LogMode.Info"/>.
```

Comment:
there is <example> tag in XML, check [link](https://habr.com/ru/post/41514/)

Reply:
Okay